app/jobs/ingest_job.rb

```diff
@@ -0,0 +1,17 @@
+class IngestJob < Hyrax::ApplicationJob
+  queue_as Hyrax.config.ingest_queue_name
+
+  after_perform do |job|
+    opts = job.arguments.reverse.detect { |x| x.is_a? Hash } || {}
```

Comment:
We want the "lastmost hash", if any.  In this case is isn't particularly important, now that the number of arguments is small (1-2), but it minimizes the number of times the `detect` block is evaluated (max once).